packages/fieldset/test/lion-fieldset.test.js

```diff
@@ -6,22 +6,38 @@ import {
   triggerFocusFor,
   triggerBlurFor,
   nextFrame,
+  defineCE,
 } from '@open-wc/testing';
 import sinon from 'sinon';
 import { localizeTearDown } from '@lion/localize/test-helpers.js';
-import '@lion/input/lion-input.js';
+import { LionInput } from '@lion/input';
 
 import '../lion-fieldset.js';
 
+const complexInputTagString = defineCE(
+  class extends LionInput {
+    parser() {
+      return this.modelValue;
+    }
+
+    formatter(mv) {
+      return mv && mv.value !== undefined ? mv.value : mv;
```

Comment:
this makes all text inputs in tests to behave like special case inputs, and according to @tlouisse this was implemented to fix some tests that contained text inputs with complex model values (❗️) and started failing after the fix

I think this is a symptom of wrong tests in the first place and those wrong tests should have been fixed instead of introducing this special cases for every single test in this file

also I think we have pretty bad practice in this test file where we reuse a template (`inputSlot` defined below) in every test while not every element defined in it is tested, I'd rather have a needed templated defined in each test separately and make tests more isolated, also preventing other issue like the one described earlier in this comment

Reply:
Fair point 👍  I will look at this later again, when I have an overview of the impact of all pending form core updates